### ISSUE TITLE
Add More JavaScript Tiny Projects

### ISSUE THREAD

**Issue by abhisheknaiidu** [abhisheknaiidu/javascript30#10]

**Comment by Yoddha99** [comment]
Hello, Can i work on this for hacktoberfest?

**Comment by Rishav0907** [comment]
Can i go for it for hacktoberfest?

**Comment by bharati-21** [comment]
Hey, @abhisheknaiidu would like to work on this as well!
Thanks and cheers!

**Comment by sanjay-xdr** [comment]
hey @abhisheknaiidu i would love to share my projects here too

**Comment by abhisheknaiidu** [comment]
Yes, @Yoddha99 @Rishav0907 @bharati-21 @sanjay-xdr Go Ahead Guys! Waiting for awesome projects to get merged!

**Comment by souraexx** [comment]
Can I work on this for Hacktoberfest and is it okay if I add two projects?

**Comment by Satyam1203** [comment]
Can I also add some tiny projects ? @abhisheknaiidu

**Comment by abhisheknaiidu** [comment]
Hey Guys, If you want you can join our Developer's Community in Discord 
Link - https://discord.gg/JKTUjj

**Comment by neupanedipen** [comment]
#28

**Comment by neupanedipen** [comment]
Hey I have opened a pull request. Can you review it?

**Comment by BRZ-OS** [comment]
Hi @abhisheknaiidu can I work on this for hacktoberfest? would like to add some project for it.

**Comment by NirajD10** [comment]
Hello, I would like to add tiny project.. kindly message me.

**Comment by Rishita-chawla** [comment]
Hi! I’d love to work on this issue.
Please assign it to me.
